crates/interpreter/src/valid.rs

```diff
@@ -412,24 +406,32 @@ struct Expr<'a, 'm> {
     is_body: bool,
     locals: Vec<ValType>,
     labels: Vec<Label<'m>>,
+    side_table: Vec<Option<SideTableEntryView>>,
+}
+
+#[derive(Debug, Default, Copy, Clone, PartialEq, Eq)]
```

Comment:
See below, doesn't need to be `Copy, PartialEq, Eq`. And probably also not `Default, Clone`, actually definitely not `Default`.

Reply:
`Copy` is used in `self.label().branches.push(branch)` for `If`.